engine/image-loader/image-loader.lisp

```diff
@@ -0,0 +1,47 @@
+(defpackage lbge.image-loader
+  (:use :cl)
+  (:export load-image make-image))
+(in-package lbge.image-loader)
+
+(defclass image ()
+  ((width :reader  width
+          :initarg :width
+          :initform nil
+          :type integer)
+   (height :reader  height
+           :initarg :height
+           :initform nil
+           :type integer)
+   (channels :reader  channels
+             :initarg :channels
+             :initform nil
+             :type string)
+   (data :reader  data
+         :initarg :data
+         :initform nil
+         :type string)))
```

Comment:
String is probably not the best type for binary data. It's usually stored as `(vector (unsiged-byte 8))`. If changed to that, other functions that work with the data will have to be changed to use that type, but it will probably open opportunities for using libraries and functions operating on binary data (which is usually stored as `(vector (unsigned-byte 8))`).